src/server/views/dashboard/macros.njk

```diff
@@ -50,85 +51,88 @@
-          <div class="govuk-radios__item">
-            <input class="govuk-radios__input" id="action-update" name="action" type="radio" value="updateNew" form="mainForm">
-            <label class="govuk-label govuk-radios__label" for="action-update">
-              Publish
-            </label>
-          </div>
-        {% endif %}
-        {% if "update-live" in actionButtons %}
-        <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="action-update" name="action" type="radio" value="updateLive" form="mainForm">
-          <label class="govuk-label govuk-radios__label" for="action-update">
-            Update live version
-          </label>
-        </div>
-        {% endif %}
-        {% if "request-changes" in actionButtons %}
-        <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="action-requestChanges" name="action" type="radio" value="requestChanges" data-aria-controls="conditional-action-edit" form="mainForm" {% if changeMessage %} checked {% endif %}>
-          <label class="govuk-label govuk-radios__label" for="action-requestChanges">
-            Request changes
-          </label>
-        </div>
-        <div class="govuk-radios__conditional" id="conditional-action-requestChanges">
-          <div class="govuk-form-group">
-            <div id="action-edit-hint" class="govuk-hint">
-              Provide simple and concise instructions to the service provider on what they need to change.
-            </div>
-            <textarea class="govuk-textarea" id="message" name="message" rows="5" aria-describedby="action-requestChanges-hint">{{ changeMessage if changeMessage }}</textarea>
-          </div>
-        </div>
-        {% endif %}
-        {% if "unpublish" in actionButtons %}
-        <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="action-unpublish" name="action" type="radio" value="unpublish" form="mainForm">
-          <label class="govuk-label govuk-radios__label" for="action-unpublish">
-            Unpublish
-          </label>
-        </div>
-        {% endif %}
-        {% if "remove" in actionButtons %}
-        <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="action-remove" name="action" type="radio" value="remove" form="mainForm">
-          <label class="govuk-label govuk-radios__label" for="action-remove">
-            Remove
-          </label>
-        </div>
-        {% endif %}
-      </div>
-    </fieldset>
-  </div>
+    {{ govukRadios({
+      name: "action",
+      fieldset: {
+        legend: {
+          text: "Select an option",
+          isPageHeading: false,
+          classes: "govuk-fieldset__legend--l"
+        }
+      },
+      items: items
+    }) }}
 
   <div class="govuk-button-group">
     {{ govukButton({
       text: "Continue",
-      classes: "govuk-button--ok govuk-!-margin-bottom-2",
-      attributes: {
```

Comment:
I don't think we need these attributes?

Reply:
I can't remember why they were there in the first place 😅
I gues it would be good to know if the positioning still looks good without this `govuk-!-margin-bottom-2`

Reply:
I haven't actually removed it - it just looks like a deletion because the end comma isn't necessary (and causes a njk compile error). see the addition on l135